spec/support/shared_examples/challenge.rb

```diff
@@ -0,0 +1,75 @@
+shared_examples_for 'a valid transition to :needs_approval' do |challenge_factory|
```

Comment:
@atareshawty mind verifying that this is correct? I might have missed something. 

Are comments mandatory?